web/styles/popovers.css

```diff
@@ -352,63 +352,56 @@ ul {
     }
 }
 
-.group-info-popover,
 .message-user-card-popover-root,
 .user-card-popover-root {
     /* 240px (width of avatar) + 2px (1px for each border) */
     width: 242px;
     padding: 0;
 }
 
-.group-info-popover {
-    .group-info-content {
-        padding: 5px 0;
-    }
-
-    .group-info {
-        padding-left: 11px;
-        padding-right: 11px;
+.popover-group-info {
+    display: flex;
+    flex-direction: column;
+    gap: 5px;
+    padding: 3px 10px;
+}
 
-        .group-name {
-            font-weight: bold;
+.popover-group-name-container {
+    display: flex;
+    align-items: flex-start;
+    gap: 5px;
+    font-size: 18px;
+    font-weight: 600;
+    line-height: 20px;
+    color: var(--color-text-full-name);
+    word-break: break-word;
+}
 
-            .zulip-icon-triple-users {
-                position: relative;
-                top: 2px;
-                margin-right: 5px;
-            }
-        }
-    }
+.popover-group-description {
+    font-size: 15px;
+    line-height: 16px;
+}
 
-    .manage-group {
-        a {
-            display: flex;
-            align-items: center;
-            padding-left: 11px;
+ul.popover-menu-member-list {
+    max-height: 300px;
+    display: flex;
+    flex-direction: column;
 
-            .zulip-icon {
-                position: relative;
-                top: -1px;
-                margin-right: 5px;
-            }
-        }
+    .simplebar-content {
+        width: unset;
     }
+}
 
-    .member-list {
-        position: relative;
-        max-height: 300px;
-        overflow-y: auto;
-        list-style: none;
-        margin-left: 0;
+.popover-menu-member {
+    display: flex;
+    align-items: center;
+    padding: 0 15px 0 5px;
+}
 
-        .bot {
-            color: hsl(180deg 5% 74%);
-            vertical-align: top;
-            width: 20px;
-            padding-top: 3.5px;
-            text-align: center;
-        }
-    }
+.popover-menu-member-name {
```

Comment:
`popover-group-menu` would be a better prefix than `popover-menu` for these, just to be consistent with the rest and clearly scoped to the group popover.

Reply:
Does sound better; Replaced the classes with the `popover-group-menu` prefix, thanks!